includes/Vendor/Vendor.php

```diff
@@ -595,9 +595,17 @@ public function get_best_selling_products() {
      * @return array
      */
     public function get_store_categories( $best_selling = false ) {
-        $transient_key = 'dokan_vendor_get_store_categories_' . $this->id;
-        if ( $best_selling ) {
-            $transient_key = 'dokan_vendor_get_best_selling_categories_' . $this->id;
+        if ( function_exists( 'wpml_get_current_language' ) ) {
```

Comment:
change only transient_key like below:
```
if ( function_exists( 'wpml_get_current_language' ) ) {
  $transient_key = 'dokan_vendor_get_store_categories_' . $current_lang . '_' . $this->id;
} else {
$transient_key = 'dokan_vendor_get_best_selling_categories_' . $this->id;
}
```
keep existing key as it is

Reply:
Code updated as per your recommendation